### DISCUSSION TITLE
Switch to SSM for api key storage?

### DISCUSSION THREAD

**Discussion by bsteiger** [TheTallMan67/Gemini-API-Functions-for-AWS#18] [Ideas]
SSM is free for all aws users, while secrets manager is not.  Might be worth it to switch over. Let me know if it's worth it to make a pull request.

https://github.com/bsteiger/Gemini-API-Functions-for-AWS/tree/feature/ssm

**Comment by TheTallMan67** [comment]
i pay 80 cents for 2 secrets as configured today (prod and sandbox), but free is good. 
Please update the readme/instructions on how to configure in AWS as well.
